mkdocs.yml

```diff
@@ -1,124 +1,207 @@
-dev_addr: '127.0.0.1:8001'
-edit_uri: edit/main/nautobot/docs
-site_dir: nautobot/project-static/docs
-site_name: Nautobot Documentation
-site_url: https://nautobot.readthedocs.io/
-repo_url: https://github.com/nautobot/nautobot
+---
+dev_addr: "127.0.0.1:8001"
+edit_uri: "edit/main/nautobot/docs"
+site_dir: "nautobot/project-static/docs"
+site_name: "Nautobot Documentation"
+site_url: "https://nautobot.readthedocs.io/"
+# TODO: replace when switching domain
+# site_url: "https://docs.nautobot.com/projects/core/"
+repo_url: "https://github.com/nautobot/nautobot"
+copyright: "Copyright &copy; The Authors"
 theme:
-    name: readthedocs
-    navigation_depth: 4
-    hljs_languages:
-        - django
-        - yaml
+  name: "material"
+  navigation_depth: 4
+  custom_dir: "docs/assets/overrides"
+  hljs_languages:
+    - "django"
+    - "yaml"
+  features:
+    - "navigation.tracking"
+    - "navigation.tabs"
+    - "navigation.tabs.sticky"
+    - "search.suggest"
+    - "search.highlight"
+    - "search.share"
+  favicon: "assets/favicon.ico"
+  logo: "assets/nautobot_logo.svg"
+  palette:
+    # Palette toggle for light mode
+    - media: "(prefers-color-scheme: light)"
+      scheme: "default"
+      primary: "black"
+      toggle:
+        icon: "material/weather-sunny"
+        name: "Switch to dark mode"
+
+    # Palette toggle for dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: "slate"
+      primary: "black"
+      toggle:
+        icon: "material/weather-night"
+        name: "Switch to light mode"
 extra_css:
-    - extra.css
+  - "assets/extra.css"
+
+# needed for RTD version flyout menu
+# jquery to be loaded by RTD conditionally - remove once merged
+# https://github.com/readthedocs/readthedocs.org/pull/9359
```

Comment:
Looks like this PR was merged 27 days ago.

Reply:
Sadly last I checked they decided to not inject jquery still. There's been some more work done it seems at removing dependency on jquery, I'll try a build without it and see if the flyout still works.

https://github.com/readthedocs/readthedocs.org/pull/9359#issuecomment-1205650777

Reply:
I tested and we still need to inject jquery... it seems RTD are working on removing their dependency on jquery, but in the meantime decided to not inject jquery together with their code.